stl/inc/format

```diff
@@ -443,13 +843,143 @@ public:
         // TODO: IDL support probably required
         _OutputIt = _It;
     }
+
+    const basic_format_args<basic_format_context>& _Get_args() const {
+        return _Args;
+    }
+};
+
+template <class _Visitor, class _Context>
+auto visit_format_arg(_Visitor&& _Vis, basic_format_arg<_Context> _Arg) -> decltype(_Vis(0)) {
+    return _STD visit(_STD forward<_Visitor>(_Vis), _Arg._Value);
+}
+
+template <class _CharT, class _OutputIt>
+_OutputIt _Write(_OutputIt _Out, monostate) {
+    _STL_INTERNAL_CHECK(false);
+    return _Out;

```

Comment:
Could this perhaps be `=delete`d?

Reply:
it does not seem to cause warnings, and we can't delete it because visit_format_arg switches based on the real, actual, runtime type of the type enumerator, resulting in the instantiation of all the alternative paths

Reply:
AFAICS users can write `std::visit_format_arg(my_visitor, basic_format_arg<SomeContextType>{})` which results in this function being called, so we should eventually have test coverage for it instead and the `_STL_INTERNAL_CHECK(false)` will have to go away in any case. I don't much care whether we remove it now vs. later, so I'll retract this comment.